internal/controllers/clusterextension_registryv1_validation_test.go

```diff
@@ -112,7 +112,7 @@ func TestClusterExtensionRegistryV1DisallowDependencies(t *testing.T) {
 			reconciler := &controllers.ClusterExtensionReconciler{
 				Client:             cl,
 				BundleProvider:     &fakeCatalogClient,
-				ActionClientGetter: acg,
+				ActionClientGetter: helmClientGetter,
```

Comment:
We probably don't need this anymore, because we are not reaching the install stage. We are just exiting after unpacked. I can try running locally and update here.

Reply:
I was just about to comment about this in the main PR.

Not now, but in a quick follow-up, we should definitely add unit tests that cover cases where `Reconcile` is called after a bundle is successfully unpacked. In those cases, we will progress to installation and need an actionClientGetter, right?

Reply:
https://github.com/operator-framework/operator-controller/issues/879